packages/backend/src/app/webhooks/webhook-controller.ts

```diff
@@ -42,6 +42,25 @@ export const webhookController: FastifyPluginAsync = async (app) => {
         },
         async (request: FastifyRequest<{ Params: WebhookUrlParams }>, reply) => {
             const flow = await getFlowOrThrow(request.params.flowId)
+            const handshakeResponse = await webhookService.handshake({
+                flow: flow,
+                payload: {
+                    method: request.method,
+                    headers: request.headers as Record<string, string>,
+                    body: await convertBody(request),
```

Comment:
Didn't want to touch too much existing code here. Since the raw request is not needed in the handler anyway I've changed it to also consume the EventPayload so it only needs to be converted once. This also removed some duplicated code from the existing code.